src/controller/python/chip/ChipReplStartup.py

```diff
@@ -77,50 +83,90 @@ def mattersetdebug(enableDebugMode: bool = True):
+        # try to adjust for convenience
+        store_path = pathlib.Path(args.trust_store)
+        if not store_path.is_absolute():
+            prefix = pathlib.Path().absolute()
+            while prefix != prefix.parent:
+                if prefix.joinpath(store_path).exists():
+                    oldpath = args.trust_store
+                    args.trust_store = prefix.joinpath(store_path).as_posix()
+                    console.print(f'''
+[bold] Replacing [/] store path {oldpath} with {args.trust_store}
+Note that you are still running from {os.getcwd()} so other relative paths may be off.
+                    ''')
+                    break
+                prefix = prefix.parent
+
+    if not os.path.exists(args.trust_store):
+        console.print(f'''
+[bold red] Missing directory:     [/]{args.trust_store}
+[bold] Your current directory: [/]{os.getcwd()}
+
+Please add a valid `--trust-store` argument to your script. If using jupyterlab,
+the command should look like:
+
+%run {{module.path}} --trust-store /chip/root/credentials/development/paa-root-certs
+
+or run `os.chdir` to the root of your CHIP repository checkout.
+        ''')
+        # nothing we can do ... things will NOT work
+        return
+
+    chip.native.Init()
+
+    global certificateAuthorityManager
+    global chipStack
+    global caList
+    global devCtrl
+
+    ReplInit(args.debug)
+
+    chipStack = ChipStack(persistentStoragePath=args.storagepath, enableServerInteractions=False)
+    certificateAuthorityManager = chip.CertificateAuthority.CertificateAuthorityManager(chipStack, chipStack.GetStorageManager())
+
+    certificateAuthorityManager.LoadAuthoritiesFromStorage()
+
+    if not certificateAuthorityManager.activeCaList:
+        ca = certificateAuthorityManager.NewCertificateAuthority()
+        ca.NewFabricAdmin(vendorId=0xFFF1, fabricId=1)
+    elif not certificateAuthorityManager.activeCaList[0].adminList:
+        certificateAuthorityManager.activeCaList[0].NewFabricAdmin(vendorId=0xFFF1, fabricId=1)
+
+    caList = certificateAuthorityManager.activeCaList
+
+    devCtrl = caList[0].adminList[0].NewController(paaTrustStorePath=args.trust_store)
+    builtins.devCtrl = devCtrl
+
+    console.print(
+        '\n\n[blue]The following objects have been created:')
+
+    console.print(
+        '''\t[red]certificateAuthorityManager[blue]:\tManages a list of CertificateAuthority instances.
+    \t[red]caList[blue]:\t\t\t\tThe list of CertificateAuthority instances.
+    \t[red]caList[n][m][blue]:\t\t\tA specific FabricAdmin object at index m for the nth CertificateAuthority instance.''')
+
+    console.print(
+        f'\n\n[blue]Default CHIP Device Controller (NodeId: {devCtrl.nodeId}): '
+        f'has been initialized to manage [bold red]caList[0].adminList[0][blue] (FabricId = {caList[0].adminList[0].fabricId}), '
+        'and is available as [bold red]devCtrl')
+
+
+main()
```

Comment:
Should this be:
```
if __name__ == "__main__":
    main()
```

Reply:
Apparently jupyter-lab does set name to main ... I did not expect that, but will make use of it. Changed!